### DISCUSSION TITLE
Plans for Expansion to other OEs?

### DISCUSSION THREAD

**Discussion by MeiMac** [Azure/ausgovcaf-cloudsoe#28] [Q&A] [answered]
This looks like a great initiative, and would assist smaller organisations to get a great baseline in their server environment security posture.
Is there any plans to expand this out of Server and do a similar project with end-user device environments (i.e. Win10)?

**Comment by lostdoco** [comment, marked as answer]
Hi @MeiMac thank you for the question 👍 

This is a viable option given the image building framework and approach and no reason the Azure Image Builder sequence couldn't be extended to End User Compute (EUC) images also for Azure Virtual Desktop (AVD) images or cloud VM workstations.

The workbook/system management story is a littler tricker however, as the device management platform for EUC is Microsoft Endpoint Manager (MEM) which doesn't run on servers, and azure update management doesn't run on EUC 😅

Having a common set of technical settings that help achieve a control shared amongst both would be the most efficient, although Windows (user) is going more down the [ MDM/CSP route](https://docs.microsoft.com/en-us/mem/intune/configuration/device-profiles) and server is tending towards writable Guest Configuration Policy (preview).
